backend/annoucementBackend.js

```diff
@@ -9,7 +9,7 @@ app.use((req, res, next) => {
     next();
 });
 
-mongoose.connect(process.env.DB_LINK, {
+mongoose.connect("mongodb+srv://plandsman:MongoDB1122!@cluster0.omjjl.mongodb.net/happyhats?retryWrites=true&w=majority", {
```

Comment:
Put this URL in your .env file as DB_LINK for the variable.

Reply:
Go ahead and check out my commit to main and read the stuff in the README for dealing with the DB.